daemon.py

```diff
@@ -0,0 +1,57 @@
+import json
```

Comment:
this was the original stand alone script I was going to have running in the background, it will work like it is and could be worth learning from but it won't play so well with django channels as I currently understand the implementation